Readme.md

```diff
@@ -24,22 +24,28 @@ Here's a trivial example:
 
 ```rust
 extern crate assert_cli;
-assert_cli::assert_cli_output("echo", &["42"], "42").unwrap();
+fn main() {
+  assert_cli::assert_cli_output("echo", &["42"], "42").unwrap();
+}
 ```
 
 Or if you'd rather use the macro:
 
 ```rust,ignore
 #[macro_use] extern crate assert_cli;
-assert_cli!("echo", &["42"] => Success, "42").unwrap();
-assert_cli!("black-box", &["--special"] => Error 42, "error no 42\n").unwrap()
+fn main() {
+  assert_cli!("echo", &["42"] => Success, "42").unwrap();
+  assert_cli!("black-box", &["--special"] => Error 42, "error no 42\n").unwrap()
+}
 ```
 
 And here is one that will fail:
 
 ```rust,should_panic
 extern crate assert_cli;
-assert_cli::assert_cli_output("echo", &["42"], "1337").unwrap();
+fn main() {
+  assert_cli::assert_cli_output("echo", &["42"], "1337").unwrap();
```

Comment:
You can remove the `should_panic` annotation if use `assert!(blabla.is_none())` instead.